libs/domains/services/feature/src/lib/service-details/service-details.tsx

```diff
@@ -100,7 +99,8 @@ export function ServiceDetails({ className, environmentId, serviceId, ...props }
   return (
     <div className={twMerge('flex flex-col shrink-0 gap-5 py-8 text-sm', className)} {...props}>
       <Section className={sectionClassName}>
```

Comment:
The bottom margin for each section isn't good, should be `20px`
<img width="381" alt="Capture d’écran 2023-10-02 à 11 34 34" src="https://github.com/Qovery/console/assets/533928/1014fccf-f93c-44b7-9bff-bacde449b744">

Reply:
Ha yes it's better!